### DISCUSSION TITLE
remove black border between window in hyprland

### DISCUSSION THREAD

**Discussion by hambosto** [kovidgoyal/kitty#7902] [Q&A] [answered]
after update, why kitty window have black border between window? how to change that? i want that black border same as background opacity.  like full transparent
![screenshot_26092024_150028](https://github.com/user-attachments/assets/2974d3f1-3443-4ee6-86a2-9b70578606a8)

**Comment by hambosto** [comment]
while I had to downgrade it and the problem was solved. just curious if there is a new setting for transparency between window in the latest version of kitty.

**Reply by hambosto** [reply]
in the previous version it was no issue with the opacity like full transparent.
![screenshot_26092024_151441](https://github.com/user-attachments/assets/f958b1be-d609-4530-91ee-65923577c1d2)

**Reply by kovidgoyal** [reply, marked as answer]
dup of https://github.com/kovidgoyal/kitty/issues/7895

**Reply by hambosto** [reply]
yeah i think i have to wait until next release. thanks
